core/frontend/src/ViewCreator3d.ts

```diff
@@ -38,6 +38,8 @@ export interface ViewCreator3dOptions {
   useSeedView?: boolean;
   /** Aspect ratio of [[Viewport]]. Required to fit contents of the model(s) in the initial state of the view. */
   vpAspect?: number;
+  /** Use the old, dimmer stylings when generating view. */
+  useV2Stylings?: boolean;
```

Comment:
I think this should be called `useDefaultLighting` and documentation should clarify what it does if true vs what it does if false/undefined.